project/Dependencies.scala

```diff
@@ -87,6 +87,7 @@ object Dependencies {
     "org.scalatest" %% "scalatest" % versions.scalatest % "test",
     "com.novocode" % "junit-interface" % versions.junitInterface % "test",
     "org.scalacheck" %% "scalacheck" % versions.scalaCheckVersion % "test",
+    "javax.xml.bind" % "jaxb-api" % "2.2.11",
```

Comment:
Can we specify the javax version in the versions block at the top, like our other pinned versions?